bin/migrate.sh

```diff
@@ -28,7 +32,7 @@ fi
 mkdir -p "$PLUGIN_DIR"
 env mv "$SRC_DIR" "$DST_DIR"
 env mv "$DST_DIR/README.md" "$PLUGIN_DIR"
-env cp "$DST_DIR/LICENSE" "$PLUGIN_DIR"
+env cp LICENSE "$PLUGIN_DIR"
```

Comment:
This file doesn't exist, my understanding is that we want to copy any of the existing license files in the repo